test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -370,7 +374,15 @@ class test_uct_cm_sockaddr : public uct_test {
 
 protected:
 
+    void skip_tcp_sockcm() {
+        if (!strcmp(GetParam()->md_name.c_str(), "tcp")) {
+            UCS_TEST_SKIP_R("tcp cm is not fully implemented");
```

Comment:
we should not use MD object for CM at all, is it possible to check against component name?

Reply:
yes, would need to query the component first.

Reply:
need to check component flags

Reply:
@yosefe why flags?
rdmacm should run all the tests and has the UCT_COMPONENT_FLAG_CM flag set while tcp should run some tests and also has this flag set.

Reply:
ok